stan/math/prim/fun/matrix_exp_action_handler.hpp

```diff
@@ -54,39 +90,66 @@ class matrix_exp_action_handler {
       A(i, i) -= mu;
```

Comment:
[optional]

You aren't changing these lines in this PR, so this is optional, but I guess these lines could just be:

```cpp
A.diagonal().array() -= mu;
```

Reply:
changed. thanks